src/domain/shared/layout/EntityPageTabs.tsx

```diff
@@ -58,7 +57,7 @@ enum NavigationActions {
 
 // https://developer.mozilla.org/en-US/docs/Web/API/Navigator/canShare
 interface ShareCapableNavigator extends Navigator {
-  canShare?(data: ShareData): boolean;
+  canShare(data?: ShareData | undefined): boolean;
```

Comment:
I was getting a weird compilation error (only in vscode) about this:
![image](https://user-images.githubusercontent.com/16032487/217857680-2aa32bc6-6af7-48ae-a66a-7bcbf112d69c.png)

looks like the definition of Navigator that comes with my vscode defines canShare this way, and typescript doesn't complain about it:
![image](https://user-images.githubusercontent.com/16032487/217858030-216db5f2-80b0-4b5e-ae7c-de7fa1a1f4fb.png)

What I haven't removed is the question mark on the call to the function, in case it's not defined in the browser:
![image](https://user-images.githubusercontent.com/16032487/217858344-a04b7b0d-1044-4978-bd75-28b44f4daf16.png)